### PR TITLE
Fixing broken model.

### LINKED ISSUE #2491: Inventory successfully created but model status is always broken after upgrading to v2.10.0
Related to #2444

### PR DESCRIPTION
**Pull request by red2k18**
Fixes #2491

### CODE REVIEW

File: google/cloud/forseti/services/dao.py [hunk 1]
```diff
@@ -608,10 +609,15 @@ def expand_special_members(cls, session):
...
-                    'project/{}'.format(project_id),
-                    roles=[role])
+                try:
+                    iam_policy = cls.get_iam_policy(
+                        session,
+                        'project/{}'.format(project_id),
+                        roles=[role])
+                    LOGGER.info('iam_policy: %s', iam_policy)
+                except NoResultFound:
+                    LOGGER.debug('Found a project in non-existent IAM binding.')
```
Comment (ahoying) [inline google/cloud/forseti/services/dao.py]: Please include parent_member in the log, and make this a warning.

File: google/cloud/forseti/services/dao.py [hunk 2]
```diff
@@ -608,10 +609,15 @@ def expand_special_members(cls, session):
                     session, '', member_types=member_type_map.keys()):
                 member_type, project_id = parent_member.split('/')
                 role = member_type_map[member_type]
-                iam_policy = cls.get_iam_policy(
-                    session,
-                    'project/{}'.format(project_id),
-                    roles=[role])
+                try:
```
Comment (ahoying) [inline google/cloud/forseti/services/dao.py]: You can add a test for this by adding a member to an iam binding in /tests/services/inventory/test_data/additional_cai_iam_policies.dump to bq_test_ds that points to a non-existent project.

'projectEditor:project_does_not_exist'

Then you'll need to regenerate the test files like this from a virtualenv that has the forseti dependencies installed:

PYTHONPATH=. python tests/services/inventory/update_cai_dumps.py
PYTHONPATH=. python tests/services/model/importer/update_test_dbs.py

Then the tests should still pass, particularly /tests/services/model/importer/importer_test.py.

To further test, update https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/tests/services/test_models.py#L57 to add a binding for projecteditor/project_does_not_exist to one of the buckets.

Then all tests in /tests/services/dao_test.py should still pass.

File: google/cloud/forseti/services/dao.py [hunk 3]
```diff
@@ -616,7 +616,8 @@ def expand_special_members(cls, session):
                         roles=[role])
                     LOGGER.info('iam_policy: %s', iam_policy)
                 except NoResultFound:
-                    LOGGER.debug('Found a project in non-existent IAM binding.')
+                    LOGGER.warning('Found a project in non-existent IAM binding'
```
Comment (ahoying) [inline google/cloud/forseti/services/dao.py]: Nit:

'Found a non-existent project, or project outside of the organization, in an IAM binding: %s'

File: tests/services/inventory/test_data/additional_cai_iam_policies.dump [hunk 4]
```diff
@@ -3,7 +3,7 @@
 # update_cai_dumps.py script. Any lines that start with a '#' are ignored.
 #
 # Bigquery Dataset with project number
-{"name":"//bigquery.googleapis.com/projects/1042/datasets/bq_test_ds","asset_type":"google.cloud.bigquery.Dataset","iam_policy":{"bindings":[{"role":"roles/bigquery.dataEditor","members":["group:test-group@forseti.testing","projectEditor:project2"]},{"role":"roles/bigquery.dataOwner","members":["projectOwner:project2","user:user@forseti.testing"]},{"role":"roles/bigquery.dataViewer","members":["allAuthenticatedUsers","domain:forseti.testing","projectViewer:project2"]}]}}
+{"name":"//bigquery.googleapis.com/projects/1042/datasets/bq_test_ds","asset_type":"google.cloud.bigquery.Dataset","iam_policy":{"bindings":[{"role":"roles/bigquery.dataEditor","members":["group:test-group@forseti.testing","projectEditor:project2"]},{"role":"roles/bigquery.dataOwner","members":["projectOwner:project2","user:user@forseti.testing"]},{"role":"roles/bigquery.dataViewer","members":["allAuthenticatedUsers","domain:forseti.testing","projectViewer:project2"]},{"role":"roles/bigquery.dataEditor","members":["group:test-group@forseti.testing","projectEditor:project_does_not_exist"]}]}}
```
Comment (ahoying) [inline tests/services/inventory/test_data/additional_cai_iam_policies.dump]: The dataEditor role already exists, you can just add this new member to the members list.

File: google/cloud/forseti/services/dao.py (hunk 3 above)
Comment (red2k18) [reply google/cloud/forseti/services/dao.py]: Done

File: tests/services/inventory/test_data/additional_cai_iam_policies.dump (hunk 4 above)
Comment (red2k18) [reply tests/services/inventory/test_data/additional_cai_iam_policies.dump]: Done, PTAL

**Pull request merged**